src/clr-angular/modal/modal-body.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive } from '@angular/core';
+
+@Directive({
+  selector: '.modal-body',
```

Comment:
Is it a good idea to tie this directive with a class name? I can see this directive being used almost all content container that has `overflow: auto` or `overflow: scroll`.

Reply:
I was thinking the same thing. The reason I used a directive was because the `.modal-body` selector is added by the app side so its not in the template of the component which is the scrollable part of the modal. Really the API needs to change where modal body can be guaranteed to exist in the modal template and we have something like `<clr-modal-body>` instead of the user using `modal-body`.

```
<clr-modal [(clrModalOpen)]="closable">
  <h3 class="modal-title">Title</h3>
  <div class="modal-body">
    ...
  </div>
  <div class="modal-footer">
    ...
  </div>
</clr-modal>
```

Reply:
Gotcha. I also asked some questions on the issue this PR is tackling. It seems like this issue is prominent in modal and wizard components so we could just proceed with this current approach.

Reply:
If this is something we need to use more broadly, we could make it more generic (either by adding more selectors or using a generic selector and apply the attribute anywhere its needed).